app/models/icon.rb

```diff
@@ -73,6 +74,12 @@ def clear_icon_ids
     UpdateModelJob.perform_later(Reply.to_s, { icon_id: id }, { icon_id: nil }, audited_user_id)
   end
 
+  def update_flat_posts
+    return unless saved_change_to_url? || saved_change_to_keyword
```

Comment:
Pretty sure that should have a question mark, which raises the question of why tests passed...

Reply:
I think Rails magic things often have functions with and without the keyword (possibly if they're using a variable with a getter that returns the same value?) but support changing it to use the ?